scripts/build.ps1

```diff
@@ -190,6 +190,10 @@ function Invoke-Build
     & $dotnetExe build $TPB_Solution --configuration $TPB_Configuration -v:minimal -p:Version=$TPB_Version -p:CIBuild=$TPB_CIBuild -p:LocalizedBuild=$TPB_LocalizedBuild
     Write-Log ".. .. Build: Complete."
 
+    if ($lastExitCode -ne 0) {
```

Comment:
Can we move the logic to check to $lastExitCode to Set-ScriptFailed function?

Reply:
Done.